### ISSUE TITLE
Pre-work Checklist: Developer: Jamie Park

### ISSUE THREAD

**Issue by jyaymie** [hackforla/website#3610]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by jyaymie** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? Under an hour

**Comment by ExperimentsInHonesty** [comment]
@jyaymie thanks so much for your updates.  You are flying through the setup and issues.  Can wait to see how you far you go!

**Comment by HackforLABot** [comment]
## Activity Log: jyaymie
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- jyaymie opened: https://github.com/hackforla/website/issues/3610 at 10/10/2022, 07:54 PM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/3610 at 10/11/2022, 08:00 PM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/2847#issuecomment-1048151464 at 10/13/2022, 02:52 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2847#issuecomment-1278219842 at 10/13/2022, 02:58 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/3610#issuecomment-1278221387 at 10/13/2022, 03:00 PM PDT
- jyaymie opened: https://github.com/hackforla/website/pull/3618 at 10/13/2022, 03:34 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3617#issuecomment-1278268404 at 10/13/2022, 04:07 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3617#issuecomment-1278276598 at 10/13/2022, 04:20 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3617#pullrequestreview-1141619595 at 10/13/2022, 04:21 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3618#issuecomment-1278377636 at 10/13/2022, 07:10 PM PDT
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3618#event-7586420260 at 10/13/2022, 07:26 PM PDT
- jyaymie issue closed by PR 3618: https://github.com/hackforla/website/issues/2847#event-7586420311 at 10/13/2022, 07:26 PM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/3201#issuecomment-1145812161 at 10/14/2022, 01:20 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/3201#issuecomment-1279426828 at 10/14/2022, 01:24 PM PDT
- jyaymie opened: https://github.com/hackforla/website/pull/3624 at 10/14/2022, 04:02 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3621#issuecomment-1279573540 at 10/14/2022, 04:07 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3621#pullrequestreview-1143091937 at 10/14/2022, 04:10 PM PDT
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3624#event-7595562563 at 10/15/2022, 03:05 PM PDT
- jyaymie issue closed by PR 3624: https://github.com/hackforla/website/issues/3201#event-7595562573 at 10/15/2022, 03:05 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3628#issuecomment-1280013456 at 10/16/2022, 10:19 AM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3628#pullrequestreview-1143319293 at 10/16/2022, 10:42 AM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/3596#issuecomment-1264691932 at 10/16/2022, 10:56 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/3596#issuecomment-1280020760 at 10/16/2022, 10:59 AM PDT
- jyaymie opened: https://github.com/hackforla/website/pull/3629 at 10/16/2022, 12:21 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3628#pullrequestreview-1144592029 at 10/17/2022, 10:57 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3633#issuecomment-1281648054 at 10/17/2022, 05:07 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3633#pullrequestreview-1144947931 at 10/17/2022, 05:23 PM PDT
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3629#event-7606983148 at 10/17/2022, 06:20 PM PDT
- jyaymie issue closed by PR 3629: https://github.com/hackforla/website/issues/3596#event-7606983198 at 10/17/2022, 06:20 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3633#pullrequestreview-1146361349 at 10/18/2022, 11:57 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3642#issuecomment-1284577969 at 10/19/2022, 02:10 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3642#pullrequestreview-1148280880 at 10/19/2022, 02:11 PM PDT
- jyaymie issue closed as completed: https://github.com/hackforla/website/issues/3610#event-7648594318 at 10/23/2022, 09:16 AM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/2600#issuecomment-1284381184 at 10/23/2022, 11:29 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2600#issuecomment-1288172342 at 10/23/2022, 11:35 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2600#issuecomment-1288471549 at 10/23/2022, 11:15 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3652#issuecomment-1289453367 at 10/24/2022, 11:49 AM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3652#pullrequestreview-1153636118 at 10/24/2022, 12:08 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2600#issuecomment-1289797521 at 10/24/2022, 05:03 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3658#issuecomment-1291166326 at 10/25/2022, 02:31 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3658#pullrequestreview-1155528625 at 10/25/2022, 02:36 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3655#issuecomment-1291172423 at 10/25/2022, 02:38 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3655#pullrequestreview-1155543091 at 10/25/2022, 02:45 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3657#issuecomment-1291190215 at 10/25/2022, 03:02 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3657#issuecomment-1291194542 at 10/25/2022, 03:08 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3659#issuecomment-1291195673 at 10/25/2022, 03:09 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3659#pullrequestreview-1155597047 at 10/25/2022, 03:14 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3657#pullrequestreview-1155610296 at 10/25/2022, 03:21 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3662#issuecomment-1291410720 at 10/25/2022, 07:27 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2600#issuecomment-1291423985 at 10/25/2022, 07:47 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3662#pullrequestreview-1155813718 at 10/25/2022, 08:32 PM PDT
- jyaymie opened: https://github.com/hackforla/website/pull/3664 at 10/26/2022, 01:06 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3664#issuecomment-1292845441 at 10/26/2022, 06:22 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3664#issuecomment-1292847914 at 10/26/2022, 06:27 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3652#pullrequestreview-1160704848 at 10/28/2022, 01:48 PM PDT
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3664#event-7710012207 at 10/31/2022, 09:25 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3687#issuecomment-1299448044 at 11/01/2022, 06:52 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3687#pullrequestreview-1164300472 at 11/01/2022, 07:09 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3686#issuecomment-1299463822 at 11/01/2022, 07:11 PM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3686#pullrequestreview-1164306571 at 11/01/2022, 07:15 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3687#issuecomment-1299469899 at 11/01/2022, 07:19 PM PDT
- jyaymie assigned: https://github.com/hackforla/website/issues/2216 at 11/01/2022, 07:29 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1299481617 at 11/01/2022, 07:35 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3689#issuecomment-1301071191 at 11/02/2022, 11:46 AM PDT
- jyaymie submitted review: https://github.com/hackforla/website/pull/3689#pullrequestreview-1165836220 at 11/02/2022, 11:50 AM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1301143213 at 11/02/2022, 12:45 PM PDT
- jyaymie unassigned: https://github.com/hackforla/website/issues/2216#issuecomment-1301143213 at 11/02/2022, 09:17 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/issues/2216#issuecomment-1301633567 at 11/02/2022, 09:41 PM PDT
- jyaymie commented on: https://github.com/hackforla/website/pull/3695#issuecomment-1305751430 at 11/07/2022, 07:07 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3695#pullrequestreview-1170576965 at 11/07/2022, 07:11 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1305765721 at 11/07/2022, 07:17 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3672#pullrequestreview-1170659650 at 11/07/2022, 07:57 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3672#issuecomment-1306138211 at 11/07/2022, 12:17 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3672#pullrequestreview-1171034427 at 11/07/2022, 12:18 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3696#issuecomment-1306200304 at 11/07/2022, 01:13 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3696#pullrequestreview-1171133019 at 11/07/2022, 01:40 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3699#issuecomment-1307760111 at 11/08/2022, 12:01 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3699#pullrequestreview-1172764494 at 11/08/2022, 12:03 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3700#issuecomment-1307764162 at 11/08/2022, 12:04 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3700#pullrequestreview-1172771505 at 11/08/2022, 12:09 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3696#pullrequestreview-1172795564 at 11/08/2022, 12:30 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3703#issuecomment-1308114353 at 11/08/2022, 06:26 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3703#pullrequestreview-1173166793 at 11/08/2022, 06:32 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3717#issuecomment-1314805315 at 11/14/2022, 09:46 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3717#pullrequestreview-1180248957 at 11/14/2022, 10:08 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3718#issuecomment-1314823031 at 11/14/2022, 10:10 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3718#pullrequestreview-1180258922 at 11/14/2022, 10:21 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3721#issuecomment-1315988412 at 11/15/2022, 03:07 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3721#pullrequestreview-1181679219 at 11/15/2022, 03:21 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3719#issuecomment-1316012450 at 11/15/2022, 03:26 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3719#pullrequestreview-1181693537 at 11/15/2022, 03:42 PM PST
- jyaymie issue closed by PR 3833: https://github.com/hackforla/website/issues/3701#event-7819340305 at 11/15/2022, 06:39 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3736#issuecomment-1324500264 at 11/22/2022, 07:18 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3736#pullrequestreview-1192185546 at 11/23/2022, 11:35 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3736#pullrequestreview-1192200951 at 11/23/2022, 11:52 AM PST
- jyaymie unassigned: https://github.com/hackforla/website/issues/2600#issuecomment-1328323817 at 11/27/2022, 11:36 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3749#issuecomment-1328533655 at 11/27/2022, 08:48 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3749#pullrequestreview-1195219727 at 11/27/2022, 09:19 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3753#issuecomment-1331317658 at 11/29/2022, 01:14 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3753#pullrequestreview-1198369424 at 11/29/2022, 01:19 PM PST
- jyaymie assigned: https://github.com/hackforla/website/issues/2393#issuecomment-1330824787 at 11/29/2022, 02:59 PM PST
- jyaymie commented on: https://github.com/hackforla/website/issues/2393#issuecomment-1331426760 at 11/29/2022, 03:01 PM PST
- jyaymie commented on: https://github.com/hackforla/website/issues/2393#issuecomment-1331620548 at 11/29/2022, 08:10 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3754#issuecomment-1355588828 at 12/16/2022, 12:50 PM PST
- jyaymie opened: https://github.com/hackforla/website/pull/3801 at 01/09/2023, 02:19 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3799#issuecomment-1376420554 at 01/09/2023, 02:28 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3799#pullrequestreview-1241295973 at 01/09/2023, 02:33 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3800#issuecomment-1376427702 at 01/09/2023, 02:34 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3800#pullrequestreview-1241303019 at 01/09/2023, 02:41 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3800#pullrequestreview-1241303833 at 01/09/2023, 02:42 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3807#issuecomment-1378156701 at 01/10/2023, 06:16 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3807#pullrequestreview-1243206471 at 01/10/2023, 06:18 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3808#issuecomment-1378158756 at 01/10/2023, 06:19 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3808#pullrequestreview-1243213798 at 01/10/2023, 06:32 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3806#issuecomment-1378174618 at 01/10/2023, 06:49 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3806#pullrequestreview-1243229117 at 01/10/2023, 07:00 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3808#issuecomment-1378193836 at 01/10/2023, 07:21 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3801#issuecomment-1378310835 at 01/10/2023, 10:53 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3801#issuecomment-1378321175 at 01/10/2023, 11:07 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3801#issuecomment-1379486772 at 01/11/2023, 01:09 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3808#pullrequestreview-1244618743 at 01/11/2023, 01:11 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3801#issuecomment-1379579390 at 01/11/2023, 02:45 PM PST
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3801#event-8207120218 at 01/11/2023, 06:39 PM PST
- jyaymie assigned: https://github.com/hackforla/website/issues/3701#event-7819340305 at 01/15/2023, 02:15 AM PST
- jyaymie commented on: https://github.com/hackforla/website/issues/3701#issuecomment-1383111960 at 01/15/2023, 02:19 AM PST
- jyaymie opened: https://github.com/hackforla/website/pull/3833 at 01/16/2023, 10:56 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3832#issuecomment-1385636670 at 01/17/2023, 07:52 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3832#pullrequestreview-1251832684 at 01/17/2023, 07:54 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3834#issuecomment-1385947709 at 01/17/2023, 11:35 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3834#pullrequestreview-1252328150 at 01/17/2023, 11:39 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3835#issuecomment-1385953978 at 01/17/2023, 11:41 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3835#pullrequestreview-1252338829 at 01/17/2023, 11:43 AM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3836#issuecomment-1385957600 at 01/17/2023, 11:44 AM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3836#pullrequestreview-1252357529 at 01/17/2023, 11:51 AM PST
- jyaymie PR was merged: https://github.com/hackforla/website/pull/3833#event-8251200847 at 01/17/2023, 05:00 PM PST
- jyaymie issue closed by PR 3833: https://github.com/hackforla/website/issues/3701#event-8251200944 at 01/17/2023, 05:00 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3833#issuecomment-1386438090 at 01/17/2023, 07:37 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3767#pullrequestreview-1252865373 at 01/17/2023, 09:51 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3940#issuecomment-1421643473 at 02/07/2023, 03:35 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3940#pullrequestreview-1288175562 at 02/07/2023, 03:44 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3954#issuecomment-1421659093 at 02/07/2023, 03:47 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3954#pullrequestreview-1288197700 at 02/07/2023, 03:54 PM PST
- jyaymie assigned: https://github.com/hackforla/website/issues/3731#issuecomment-1411583859 at 02/07/2023, 03:56 PM PST
- jyaymie unassigned: https://github.com/hackforla/website/issues/3731#issuecomment-1411583859 at 02/07/2023, 03:56 PM PST
- jyaymie commented on: https://github.com/hackforla/website/pull/3947#issuecomment-1421666661 at 02/07/2023, 03:57 PM PST
- jyaymie submitted review: https://github.com/hackforla/website/pull/3947#pullrequestreview-1288237082 at 02/07/2023, 04:17 PM PST
